From ad10920202ef61950d39caeddbeb80c745be0283 Mon Sep 17 00:00:00 2001
From: "shuicheng.peng" <shuicheng.peng@amlogic.com>
Date: Fri, 20 Jun 2014 13:25:13 +0800
Subject: [PATCH 4371/5965] PD#93407: fix blue screen with only audio output

fix blue screen with only audio output when power on at DTV.
video channel will free when set pcrid, then no video dmx output,add pcr_chan initial
---
 drivers/amlogic/dvb_tv/aml_dvb.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/amlogic/dvb_tv/aml_dvb.c b/drivers/amlogic/dvb_tv/aml_dvb.c
index 41a51046d153..c776732660a7 100755
--- a/drivers/amlogic/dvb_tv/aml_dvb.c
+++ b/drivers/amlogic/dvb_tv/aml_dvb.c
@@ -205,6 +205,7 @@ static int aml_dvb_dmx_init(struct aml_dvb *advb, struct aml_dmx *dmx, int id)
 	dmx->aud_chan = -1;
 	dmx->vid_chan = -1;
 	dmx->sub_chan = -1;
+	dmx->pcr_chan = -1;
 
 	if ((ret = aml_dmx_hw_init(dmx)) <0) {
 		pr_error("demux hw init error %d", ret);
-- 
2.19.0

